pgbouncer/tests/conftest.py

```diff
@@ -27,12 +27,15 @@ def container_up(service_name, port):
 @pytest.fixture(scope="session")
 def dd_environment():
     """
-    Start postgres and install pgbouncer. If there's any problem executing
-    docker-compose, let the exception bubble up.
+    Start postgres and install pgbouncer. If there's any problem executing docker-compose, let the exception bubble up.
     """
+    compose_file = 'docker-compose.yml'
+    version = common.get_version_from_env()
+    if int(version[1]) < 10:
```

Comment:
(nit) maybe we can use `from packaging import version` for version comparison ?

Example:

https://github.com/DataDog/integrations-core/blob/666f871e6dc6f42d68aa5b4b954f96813de0d789/fluentd/tests/util.py#L5-L12